deploy/dev/package.json

```diff
@@ -3,7 +3,9 @@
   "description": "Docker compose setup for dev environment",
   "type": "module",
   "scripts": {
-    "dev": "saf-docker generate && docker compose --env-file .env.dev -f docker-compose.yaml up"
+    "dev": "process-compose up -f process-compose.yaml -e .env.dev",
+    "dev:with-monitoring": "process-compose up -f process-compose.yaml -f process-compose.monitoring.yaml -e .env.dev",
+    "dev:monitoring-only": "process-compose up -f process-compose.monitoring.yaml -e .env.dev"
```

Comment:
Decided to split out the monitoring stuff; in most cases I wouldn't need everything running.